extra/html-extra/Cva.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Extra\Html;
+
+/**
+ * Class Variant Authority (Cva) resolver.
+ *
+ * The Cva concept is used to render multiple variations of components, applying
+ * a set of conditions and recipes to dynamically compose CSS class strings.
```

Comment:
I'd use this in the documentation, instead of the front-end / shadcn paragraph

Reply:
Let's remove this paragraph.